.github/workflows/build.yml

```
@@ -0,0 +1,106 @@
+        with:
+          path: packages
+          key: ${{ runner.os }}-${{env.BUILD_PLATFORM}}-${{env.BUILD_CONFIGURATION}}-${{ hashFiles('**/packages.config') }}
+
+      - name: Restore NuGet packages
+        if: steps.skip_check.outputs.should_skip != 'true'
+        working-directory: ${{env.GITHUB_WORKSPACE}}
+        run: |
+          nuget restore ${{env.SOLUTION_FILE}}
+
+      - Export program info
+        if: steps.skip_check.outputs.should_skip != 'true'
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 96: mapping values are not allowed in this context

(syntax-check)